src/cursor.rs

```diff
@@ -188,9 +190,10 @@ impl Row {
     }
 
     pub fn iter(&self) -> impl Iterator<Item = (&'_ str, &'_ Value)> + use<'_> {
-        self.column_mapping
+        self.column_names
             .iter()
-            .map(|x| (x.0.as_str(), &self.values[*x.1]))
+            .map(|x| (x, self.column_mapping[x]))
```

Comment:
Please call it `index` for clarity. And maybe the two `map`s can be combined into one?

Reply:
I used `column_name` because I thought it was even clearer. `map`s are now combined